medical_prescription_sale_stock_us/models/medical_prescription_order_line.py

```diff
@@ -31,27 +30,31 @@ class MedicalPrescriptionOrderLine(models.Model):
         compute='_compute_can_dispense_and_qty',
         help='Total units allowed in the prescription, including refills',
     )
-    last_dispense_remain_qty = fields.Float(
+    dispense_remain_qty = fields.Float(
         string='Dispense Remaining Qty',
         compute='_compute_dispense_remain',
-        help='Estimated number of units remaining from last dispense',
-    )
-    last_dispense_remain_percent = fields.Float(
-        string='Dispense Remaining Percent',
-        compute='_compute_dispense_remain',
-        help='Estimated percentage remaining from last dispense',
+        help='Estimated number of units remaining from all finished dispenses',
     )
-    last_dispense_remain_day = fields.Float(
+    dispense_remain_day = fields.Float(
         string='Dispense Remaining Days',
         compute='_compute_dispense_remain',
-        help='Estimated days remaining from last dispense',
+        help='Estimated days remaining based on all finished dispenses',
     )
 
     @api.multi
+    @api.depends(
```

Comment:
Hmm. At the time, this seemed to fix an issue in one of the tests that looked like it was being caused by stale data, but I just removed the wrapper and everything is still passing, so it must have been extraneous all along.